### DISCUSSION TITLE
How do I obtain the an auction league draft info for basketball?

### DISCUSSION THREAD

**Discussion by ZenHWu** [cwendt94/espn-api#329] [ESPN API]
I've been trying to use league.draft but it only returns a blank list.
![image](https://user-images.githubusercontent.com/104530554/166161803-8adaafb1-6993-4f73-aee5-08144cee044a.png)

**Comment by cwendt94** [comment]
currently basketball does not have the draft feature implemented. I will create a ticket so that it can be added #331
